project/Settings.scala

```diff
@@ -142,7 +93,7 @@ object Settings {
     val geminiLocales           = "0.1.0-2019a"
 
     // Gemini Libraries
-    val gspMath                 = "0.1.1"
+    val gspMath                 = "0.1.2"
```

Comment:
Change this back to 0.1.1 … the 0.1.2 jarfile is empty for some reason. Will figure that out today.

Reply:
Ok try 0.1.3 … no idea what happened